### DISCUSSION TITLE
OAS 3.1.0 support

### DISCUSSION THREAD

**Discussion by n2ygk** [encode/django-rest-framework#7762] [Ideas & Suggestions]
OpenAPI Specification Release 3.1.0 has been released. There may be opportunities for improvements or possible breaking changes. 

The specification: http://spec.openapis.org/oas/v3.1.0

The release notes are unfortunately not all in one place and may not fully described all the changes:

- https://github.com/OAI/OpenAPI-Specification/releases/tag/3.1.0-rc0
- https://github.com/OAI/OpenAPI-Specification/releases/tag/3.1.0-rc1
- https://github.com/OAI/OpenAPI-Specification/releases/tag/3.1.0

**Comment by auvipy** [comment]
related https://www.buymeacoffee.com/relequestual/json-schema-2020-12-released
